apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -66,6 +66,10 @@ const VersionHistoryDropdown: React.FunctionComponent<
     state => state.lab2Project.viewingOldVersion
   );
 
+  // If this is a teacher viewing a student's project, we hide the restore button,
+  // but still allow viewing old versions.
+  const viewAsUserId = useAppSelector(state => state.progress.viewAsUserId);
```

Comment:
Is there historical context on this variable name? `viewAsUserId` = "view as a teacher" is unintuitive to me, though I'm probably missing context :)

Reply:
Yes, we debated the name and this was the best we came up with: https://github.com/code-dot-org/code-dot-org/pull/58269/files/18b375a99d27398a7cdbfb9d61fd9a102dab2636#r1583489705